frontend/src/App.js

```diff
@@ -214,6 +216,7 @@ class App extends Component {
             sessionId={this.state.sessionId}
             logout={this.logout}
             userAttributes={this.state.userAttributes}
+            fetchRequest={this.fetchRequest}
```

Comment:
Instead of passing it down to a component, hoist it out to a file in a `utils` directory, and import it to the components you use it in. This maximises your code reuse, and keeps it DRY